src/corehost/common/trace.cpp

```diff
@@ -122,7 +123,18 @@ void trace::error(const pal::char_t* format, ...)
     // Always print errors
     va_list args;
     va_start(args, format);
-    pal::err_vprintf(format, args);
+
+    if (g_error_writer == nullptr)
+    {
+        pal::err_vprintf(format, args);
+    }
+    else
+    {
+        pal::char_t buffer[1024];
```

Comment:
If we use this for backtraces, 1K might not be enough.  I'm OK leaving it as is...

Reply:
Forgot to get back to this.. now it should work for any size.